dags/search_rollup.py

```diff
@@ -24,25 +24,13 @@
 }
 
 
-dag_daily = DAG('search_rollup_daily',
-                 default_args=default_args,
-                 schedule_interval='@daily')
-
-
 dag_monthly = DAG('search_rollup_monthly',
                   default_args=default_args,
-                  schedule_interval='@monthly')
-
+                  schedule_interval='0 0 2 * *')
```

Comment:
Why change the interval here?

Reply:
We have to wait for data on the last day of the month to be processed before running the monthly job. The failure mode when we're missing a few days of data in a monthly rollup is subtle and currently not validated -- it would require asserting that the rows for all days of the current month exist. 

This could be offset from UTC+0 a few hours to account for `main_summary` running.